etc/avocado/avocado.conf

```diff
@@ -19,6 +19,8 @@ enabled = True
 installed_packages = False
 # Whether to run certain commands in bg to give extra job debug information
 profiler = False
+# Force LANG for sysinfo collection
+lang = C
```

Comment:
In light of my other comment about LANG vs LC_ALL, I suggest `locale` instead of `lang` as the name for this variable.